projects/client/src/app/pages/home/Home.tsx

```diff
@@ -1,81 +1,74 @@
 import { useEffect, useState } from 'react';
 
+import { getAllProducts, ProductApi } from '@/api/productApi';
 import ProductCard from '@/app/components/product-Card/ProductCard';
 
 import carBanner from './assets/car.png';
-import jsonData from './assets/data.json';
 import leftIcon from './assets/left.png';
 import rightIcon from './assets/right.png';
 
-interface Product {
-  id: string;
-  categoryId: string;
-  name: string;
-  description: string;
-  brand: string;
-  price: string;
-  image: string;
-  stock: string;
-  discount: string;
-  unitsold: string;
-}
-
 function Home() {
   const [brands, setBrands] = useState<Set<string>>(new Set());
-  const [offersData, setOffersData] = useState<Product[]>([]);
-  const [bestSellersData, setBestSellersData] = useState<Product[]>([]);
+  const [offersData, setOffersData] = useState<ProductApi[]>([]);
```

Comment:
Why are they changing the name?

Reply:
We changed the Product[] to ProductApi[] because we are using the interface from the productApi instead of having 2 identical ones

Reply:
makes sense to me, but I prefer product, since is a Product right?